CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ else()
   set(CMAKE_CXX_FLAGS_DEFAULT "")
 endif()
 
+find_package(Threads REQUIRED)
```

Comment:
I need to call `find_package` before any call to `PCL_ADD_EXAMPLE`, `PCL_ADD_LIBRARY`, ... but yeah, I could move to the other dependencies.

Reply:
Update to code to fix it. There is only one question open: what do you think about `THREADS_PREFER_PTHREAD_FLAG` option? Should be set it or not?

Btw: `find_package` is required, because Unix requires threading library and on Windows `Threads::Threads` will be always an empty target, so MSVC doesn't complain that there is no threading library on Windows.

Reply:
> Updated code to fix it. There is only one question open: what do you think about `THREADS_PREFER_PTHREAD_FLAG` option? Should be set it or not?

Sorry. Totally overlooked this question. I have no reason to be opinionated here so I would not set anything.